modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -2361,8 +2361,7 @@ export const console: ConsoleNS = {
                 },
                 myaccount: {
                     description: "Self-service portal for your users.",
-                    popup: "Share this link with your users to allow access to My Account" +
-                    " and to manage their accounts.",
+                    popup: "Share this link with your users to allow access to My Account.",
```

Comment:
Shall we update sinhala and french translates as well to be tally with this.

Reply:
Fixed with 1d895ae4f06210a4ffaca5424c0e898c4047c969